solutions/0000-0099/0003-longest-substring-without-repeating-characters-medium.md

```diff
@@ -116,6 +116,42 @@ class Solution {
 }
 ```
 </TabItem>
+
+<TabItem value="javascript" label="JavaScript">
+<SolutionAuthor name="@MithunPrabhu777"/>
+
+```javascript
+var lengthOfLongestSubstring = function (s) {
+    const letterCountMap = new Map();
+    
+    let [left,right,max] = [0,0,0];
+    
+    while(right < s.length){
```

Comment:
Too many empty lines, leave empty line wherever it's needed. 
Add comments for each line and why we are doing this operation..

Reply:
Drafting changes